src/plugins/intel_gpu/src/kernel_selector/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ add_custom_command(OUTPUT "${CODEGEN_INCDIR}/${PRIM_DB}"
 
 add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${CODEGEN_CACHE_SOURCES})
 
+if(NOT BUILD_SHARED_LIBS)
+    target_compile_definitions(${TARGET_NAME} PUBLIC OPENVINO_STATIC_LIBRARY)
```

Comment:
generator expression?

Reply:
I try to fix windows static build (on my local machine all passed :) )